src/main/webapp/style.css

```diff
@@ -7,3 +7,57 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.listing-container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.listing-container h3.title {
+  border-bottom: 1px solid grey;
+  font-size: 40px;
+  text-align: center;
+}
+
+.listing-container .listing {
+  border-radius: 8px;
+  box-shadow: 0 4px 16px -8px grey;
+  display: flex;
+  margin: 32px 0;
+}
+
+.listing .listing-left {
+  align-items: center;
+  background-image: url("images/Food_Example.jpg") no-repeat center center fixed;
+  border-radius: 8px 0 0 8px;
+  display: flex;
+  justify-content: center;
+  min-width: 82px;
+  padding: 8px 48px;
+}
+
+.listing .listing-right {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0 24px;
+}
+
+.listing .listing-right h3.listing-title {
+  color: red;
+  font-size: 24px;
+  margin: 24px 0 10px 0;
+  text-transform: uppercase;
+}
+
+.listing .listing-right .listing-chat {
+  align-items: center;
+  background: yellow;
+  border-radius: 16px;
+  display: flex;
+  font-size: 14px;
+  justify-content: center;
+  margin: 24px 0;
+  padding: 8px;
+  width: 100px;
+}
```

Comment:
I would say that maybe you should create a separate file that says something like "card_style.css" and add this code to that new file.